android/src/main/java/nordpol/TagDispatcher.java

```diff
@@ -153,7 +158,52 @@ private void disableForegroundDispatch(NfcAdapter adapter) {
         adapter.disableForegroundDispatch(activity);
     }
 
+    static byte[] requireStatus(byte[] receivedApdu, byte[] expectedApdu) throws IOException {
```

Comment:
Yeah, that is a bit tricky. maybe you could add a function trancesiveAndRequireStatus(...) that does exactly that but takes the IsoCard and does the transceive? And then the status function could be hasStatus() and return boolean?